src/main/kotlin/com/ampnet/payoutservice/config/JsonConfig.kt

```diff
@@ -17,18 +17,7 @@ class JsonConfig {
     @Primary
     fun objectMapper(): ObjectMapper {
         val mapper = ObjectMapper()
-        mapper.propertyNamingStrategy = PropertyNamingStrategies.SnakeCaseStrategy()
-        mapper.registerModule(JavaTimeModule())
-        mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
-        mapper.configure(DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES, false)
-        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-        return mapper.registerModule(KotlinModule.Builder().build())
-    }
-
-    @Bean
-    fun camelCaseObjectMapper(): ObjectMapper {
-        val mapper = ObjectMapper()
-        mapper.propertyNamingStrategy = PropertyNamingStrategies.LowerCaseStrategy()
+        mapper.propertyNamingStrategy = PropertyNamingStrategies.LowerCamelCaseStrategy()
```

Comment:
It would be good to use `SnakeCase` because all other services have snake as default for our frontend.